src/content/docs/apm/agents/php-agent/other-integrations/magento-business-insights.mdx

```diff
@@ -0,0 +1,263 @@
+                        - name: MYSQLLoginsPerDay
+                        run: SELECT DATE(last_visit_at) as 'Date', count(*) as 'Total Logins', count(DISTINCT customer_id) AS 'Number of Customers' FROM `customer_visitor` GROUP BY customer_id, DATE(last_visit_at);
+                        - name: MYSQLBestSellersDaily
+                        run: SELECT SUM(qty_ordered) as 'QuantityOrdered', product_name FROM sales_bestsellers_aggregated_daily GROUP BY product_id, product_name ORDER BY QuantityOrdered DESC;
+                        - name: MYSQLRevenuePerMonthAndYear
+                        run: SELECT SUM(grand_total) AS 'TotalRevenue', AVG(grand_total) AS 'AverageRevenue', YEAR(created_at) AS 'Year', MONTH(created_at) AS 'Month' FROM sales_order WHERE status = 'complete' GROUP BY YEAR(created_at), MONTH(created_at);
+                        - name: MYSQLMostViewedProductsDaily
+                        run: SELECT SUM(views_num) as 'viewCount', product_name FROM report_viewed_product_aggregated_daily GROUP BY product_name ORDER BY viewCount DESC;
+                        - name: MYSQLConversionRate
+                        run: WITH cte1 AS (SELECT soi.product_id AS 'SalesProductId', soi.name AS 'ProductName', COUNT(soi.order_id) AS 'totalOrders' FROM `sales_order_item` soi group by soi.product_id, soi.name), cte2 AS (SELECT viewed.product_id AS 'productId', COUNT(viewed.product_id) AS 'totalViews' FROM `report_viewed_product_index` viewed group by viewed.product_id) SELECT SalesProductId as 'productID', ProductName, totalViews,totalOrders,totalOrders/totalViews *100 as 'conversionRateInPercent' FROM cte1 JOIN cte2 WHERE cte1.SalesProductId = cte2.ProductId;
+                        - name: MYSQLTopTenSalesOrdersByValue
+                        run: SELECT entity_id AS 'orderId', total_item_count AS 'orderQuantity', created_at AS 'createdDate', grand_total AS 'grandTotal' FROM sales_order ORDER BY grand_total DESC LIMIT 10;
+                        - name: MYSQLOrdersAverage
+                        run: SELECT AVG(order_count) AS 'AverageNumberOfOrders', Year AS 'Year' FROM (SELECT YEAR(created_at) AS 'Year', COUNT(DISTINCT order_id) AS order_count FROM sales_order_item GROUP BY MONTH(created_at), YEAR(created_at)) subquery GROUP BY Year;
+                        - name: MYSQLProductsTotal
+                        run: SELECT count(*) AS 'Total Products' FROM catalog_product_entity_varchar WHERE attribute_id = 73;
+                        custom_attributes: # can apply additional at a nested level
+                            attype: blocking-query
+                            host: CustomMagentoMySQL
+                ```
+
+        3. [Restart the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/manage-your-agent/start-stop-restart-infrastructure-agent/).
+    
+    </Step>
+
+<Step>
+
+    ## Install the Magento business insights dashboard [#find-data]
```

Comment:
we treat 'Magento Business Insights' as title case elsewhere, probably need to make it sentence case everywhere